### PR TITLE
search: Update search context integration tests to test CM implementation

### PR DESCRIPTION
**Pull request by fkling**
This commit updates those tests (well, one of them), to also test the
CodeMirror implementation.

## Test plan

Integration test tests CodeMirror implementation and passes.

## App preview:

- [Web](https://sg-web-fkling-cm-search-context.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-pytxajgpyg.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/enterprise/searchContexts/SearchContextForm.tsx [hunk 1]
```diff
@@ -396,9 +396,8 @@ export const SearchContextForm: React.FunctionComponent<React.PropsWithChildren<
                 <div className={classNames('mt-3', styles.searchContextFormVisibility)}>
                     <div className="mb-3">Visibility</div>
                     {visibilityRadioButtons.map((radio, index) => (
-                        <>
+                        <React.Fragment key={radio.visibility}>
```
Comment (fkling) [inline client/web/src/enterprise/searchContexts/SearchContextForm.tsx:399]: I added this to remove a React warning about "list elements" needing a `key` prop.

File: client/web/src/integration/search.test.ts [hunk 2]
```diff
@@ -408,8 +408,6 @@ describe('Search', () => {
             await driver.page.waitForSelector('[data-testid="search-result-match-code-excerpt"] .match-highlight', {
                 visible: true,
             })
-            await driver.page.waitForSelector('#monaco-query-input', { visible: true })
```
Comment (fkling) [inline client/web/src/integration/search.test.ts:411]: I doesn't look like those are actually needed anymore.

File: client/web/src/integration/search-contexts.test.ts [hunk 3]
```diff
@@ -147,15 +108,32 @@ describe('Search contexts', () => {
...
+            })
+
+            const editor = createEditorAPI(driver, editorName, editorSelector)
+
+            await driver.page.goto(
+                driver.sourcegraphBaseUrl + '/search?q=context:%40unavailableCtx+test&patternType=regexp',
+                { waitUntil: 'networkidle0' }
+            )
+            await driver.page.waitForSelector('.test-selected-search-context-spec', { visible: true })
+            await editor.waitForIt()
```
Comment (limitedmage) [inline client/web/src/integration/search-contexts.test.ts:133]: https://www.youtube.com/watch?v=k9AyO8h2I0k

**Review by limitedmage — APPROVED** [review]

File: client/web/src/integration/search-contexts.test.ts (hunk 3 above)
Comment (fkling) [reply client/web/src/integration/search-contexts.test.ts:133]: Now I want first class support for music comments :D

**Pull request merged**